runtime-decompiler/src/main/java/org/jrd/backend/data/ArchiveManager.java

```diff
@@ -0,0 +1,58 @@
+package org.jrd.backend.data;
+
+import org.jrd.backend.communication.FsAgent;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+public class ArchiveManager {
+    File c;
+    String tmpdir = System.getProperty("java.ioo.tmpdir");
+    ArrayList<String> currentJars = new ArrayList<>();
+
+    public ArchiveManager(File c) {
+        this.c = c;
+    }
+
+    public boolean isClassInFile(String clazz) throws IOException {
+        ZipInputStream zis = new ZipInputStream(new FileInputStream(c));
+        if (!currentJars.isEmpty()) {
+            currentJars = new ArrayList<>();
+        }
+        currentJars.add(c.getName());
+        var ret = findClazz(zis, clazz);
```

Comment:
Please use boolean here....
varhave meaning only in two cases - when it is var a = new SomeObject() so you can really imediatley see the type, or if the type is some.really.Terrible.Nested.hidden.scary.Class

Reply:
I was only testing if it actually works as I learned about it in class. This is fixed by a commit later when I noticed I accidentally left it in.